public/javascripts/gformFileUpload.js

```diff
@@ -32,11 +32,28 @@
 
   // Set up event handlers etc
   function init() {
-    $('button[name$="-uploadButton"]').css("display", "none")
     $(".govuk-file-upload").on("change", handleFileUpload);
+    $('button[name$="multiFile"]').on("click", handleEmptyFileUploadMulti);
     $('button[name$="singleFile"]').on("click", handleEmptyFileUpload);
   }
 
+   function handleEmptyFileUploadMulti(e) {
+     e.preventDefault();
+     const input = $(".govuk-file-upload")
+     var inputFile = input[0];
+     if (inputFile.files.length === 0) {
+       const id = input.attr("id");
+       const errorRedirect = $("#gf-upscan-" + id).find('input[name="error_action_redirect"]').val();
+       const params = {
+           errorMessage: "Missing file",
```

Comment:
Can we add an error message for welsh? 
`    errorMessage: {
      en: "Missing file",
      cy: "..."
    },`